app/Filament/Resources/GameHashResource.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources;
+
+use App\Filament\Extensions\Resources\Resource;
+use App\Filament\Resources\GameHashResource\Pages;
+use App\Models\GameHash;
+use Filament\Forms\Form;
+use Filament\Support\Enums\FontFamily;
+use Filament\Tables;
+use Filament\Tables\Table;
+
+class GameHashResource extends Resource
+{
+    protected static ?string $model = GameHash::class;
+
+    protected static ?string $navigationIcon = 'fas-file-archive';
+
+    protected static ?string $navigationGroup = 'Platform';
+
+    protected static ?int $navigationSort = 2;
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                // TODO
+            ]);
+    }
+
+    // TODO link values to respective pages in Filament once created
+    public static function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                Tables\Columns\TextColumn::make('md5')
+                    ->label('MD5')
+                    ->searchable()
+                    ->fontFamily(FontFamily::Mono)
+                    ->url(fn (GameHash $record): string => route('game.hash.manage', ['game' => $record->game])),
+
+                Tables\Columns\TextColumn::make('game.title')
+                    ->label('Game')
+                    ->searchable()
+                    ->url(fn (GameHash $record): string => route('game.show', ['game' => $record->game])),
+
+                Tables\Columns\TextColumn::make('user.display_name')
+                    ->url(fn (GameHash $record): string => route('user.show', ['user' => $record->user])),
```

Comment:
> Missing required parameter for [Route: user.show] [URI: user/{user}] [Missing parameter: user].

When searching on "monopoly". Many `game_hashes` records have a NULL user_id (were linked before we started tracking)

Reply:
Good catch, thanks. Resolved in latest.